src/openrct2-ui/windows/TileInspector.cpp

```diff
@@ -1884,7 +1884,8 @@ static void window_tile_inspector_paint(rct_window* w, rct_drawpixelinfo* dpi)
                 auto ride = get_ride(rideId);
                 if (ride != nullptr)
                 {
-                    auto rideType = RideTypeDescriptors[ride->type].Naming.Name;
+                    const auto& rtd = ride->GetRideTypeDescriptor();
+                    auto rideType = rtd.Naming.Name;
```

Comment:
```suggestion
                    auto rideType = ride->GetRideTypeDescriptor().Naming.Name;
```

Reply:
Is rideType really a good name for this surely rideTypeName or typeName would be better. rideType generally means the type used to get the rtd.